editor/test/effects.js

```diff
@@ -281,6 +281,8 @@ describe( 'effects', () => {
 	} );
 
 	describe( '.REQUEST_POST_UPDATE_SUCCESS', () => {
+		const handler = effects.REQUEST_POST_UPDATE_SUCCESS;
+
 		beforeAll( () => {
 			selectors.getDirtyMetaBoxes = jest.spyOn( selectors, 'getDirtyMetaBoxes' );
```

Comment:
My IDE complains that `selectors.getDirtyMetaBoxes` is a constant and shouldn't be overridden.